src/Themes/BootstrapTheme/Views/Layout.liquid

```diff
@@ -7,7 +7,10 @@
 
```

Comment:
`Layout.liquid` can be deleted. These are base templates for a theme and they're not needed for a module.